secrets.js

```diff
@@ -31,13 +35,18 @@ function getStaticSecret(api, name, variableName, akeylessToken, exportSecretsTo
             'token': akeylessToken,
             'names': [name],
         })).then(staticSecret => {
+            // Mask secret value in output
+            var secretValue = staticSecret[name];
```

Comment:
var -> const, e.g.:

```
const secretValue = staticSecret[name];
```

Ref: https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa

Reply:
Agreed, constant is better here (var was just memory muscle).